<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindowClass</class>
 <widget class="QMainWindow" name="MainWindowClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>850</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
	color: #aeaeae;
	font-family: &quot;Roboto&quot;;
}

#content,
#optionsContent,
#installContent {
	background: #2d2d30;
	background: rgb(30, 30, 38);
}

#caption {
	background: #2d2d34;
	background: rgb(32, 32, 38);
}

#footer,
#optionsFooter,
#installFooter {
	background: #202023;
	background: rgb(32, 32, 38);
	max-height: 50px;
	min-height: 50px;
}

QCheckBox {
    spacing: 5px;
}

QCheckBox:hover {
	color: #fff;
}

QCheckBox::indicator {
	image: none;
	background: #2a2a2a;
	border: 2px solid#2a2a2a;
}

QCheckBox::indicator:unchecked:hover {
	background: #3a3a3a;
	border: 2px solid#3a3a3a;
}

QCheckBox::indicator:checked {
	background: rgb(62, 204, 156);
	border: 2px solid#3a3a3a;
}
</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>16777215</height>
    </size>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>2</number>
    </property>
    <property name="topMargin">
     <number>2</number>
    </property>
    <property name="rightMargin">
     <number>2</number>
    </property>
    <property name="bottomMargin">
     <number>2</number>
    </property>
    <item>
     <widget class="QWidget" name="caption" native="true">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>100</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>100</height>
       </size>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>6</number>
       </property>
       <property name="leftMargin">
        <number>60</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="icon">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="maximumSize">
          <size>
           <width>40</width>
           <height>40</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap resource="resources.qrc">:/images/logoPtb</pixmap>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
         <property name="textInteractionFlags">
          <set>Qt::NoTextInteraction</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="font">
          <font>
           <family>Roboto</family>
           <pointsize>30</pointsize>
          </font>
         </property>
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#3ecc9c;&quot;&gt;Better&lt;/span&gt;&lt;span style=&quot; color:#ffffff;&quot;&gt;Discord&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>265</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QWidget" name="widget" native="true">
         <property name="styleSheet">
          <string notr="true">QPushButton {
	background: transparent;
}</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item alignment="Qt::AlignRight|Qt::AlignTop">
           <widget class="QPushButton" name="captionHelp">
            <property name="minimumSize">
             <size>
              <width>30</width>
              <height>24</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>30</width>
              <height>24</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Roboto</family>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton {
	border-image: url(:/images/qNormal);
}

QPushButton:hover {
	border-image: url(:/images/qHover);
}</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item alignment="Qt::AlignRight|Qt::AlignTop">
           <widget class="QPushButton" name="captionMin">
            <property name="minimumSize">
             <size>
              <width>30</width>
              <height>24</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>30</width>
              <height>24</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Roboto</family>
              <pointsize>13</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton {
	border-image: url(:/images/mNormal);
}

QPushButton:hover {
	border-image: url(:/images/mHover);
}</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item alignment="Qt::AlignRight|Qt::AlignTop">
           <widget class="QPushButton" name="captionClose">
            <property name="minimumSize">
             <size>
              <width>30</width>
              <height>24</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>30</width>
              <height>24</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Roboto</family>
              <pointsize>13</pointsize>
              <weight>50</weight>
              <bold>false</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton {
	border-image: url(:/images/xNormal);
}

QPushButton:hover {
	border-image: url(:/images/xHover);
}</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QStackedWidget" name="mainStack">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="pageInitial">
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QWidget" name="content" native="true">
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item alignment="Qt::AlignTop">
            <widget class="QWidget" name="installedProducts" native="true">
             <layout class="QVBoxLayout" name="verticalLayout_8">
              <property name="spacing">
               <number>20</number>
              </property>
              <property name="leftMargin">
               <number>0</number>
              </property>
              <property name="topMargin">
               <number>0</number>
              </property>
              <property name="rightMargin">
               <number>0</number>
              </property>
              <property name="bottomMargin">
               <number>0</number>
              </property>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QWidget" name="prodcutsSplit" native="true">
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>20</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">border-bottom: 1px solid rgb(62, 204, 156);
color: rgb(62, 204, 156);
margin-left: 65px;</string>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_2">
              <property name="spacing">
               <number>0</number>
              </property>
              <property name="leftMargin">
               <number>0</number>
              </property>
              <property name="topMargin">
               <number>0</number>
              </property>
              <property name="rightMargin">
               <number>0</number>
              </property>
              <property name="bottomMargin">
               <number>0</number>
              </property>
              <item>
               <widget class="QLabel" name="label_3">
                <property name="enabled">
                 <bool>true</bool>
                </property>
                <property name="font">
                 <font>
                  <family>Roboto</family>
                  <pointsize>10</pointsize>
                  <kerning>true</kerning>
                 </font>
                </property>
                <property name="text">
                 <string>Available Products</string>
                </property>
                <property name="indent">
                 <number>0</number>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item alignment="Qt::AlignTop">
            <widget class="QWidget" name="availableProducts" native="true">
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>16777215</height>
              </size>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_4">
              <property name="spacing">
               <number>20</number>
              </property>
              <property name="leftMargin">
               <number>0</number>
              </property>
              <property name="topMargin">
               <number>0</number>
              </property>
              <property name="rightMargin">
               <number>0</number>
              </property>
              <property name="bottomMargin">
               <number>0</number>
              </property>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="footer" native="true">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>50</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>50</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton {
	background: transparent;
    color: #fff;
    padding: 7px 10px;
    min-width: 50px;
	background: #17171c;
	border-radius: 5px;
}

QPushButton::hover {
	background: rgb(35, 35, 38);
	background: rgb(62, 204, 156)
}</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_9">
           <item>
            <spacer name="horizontalSpacer_5">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>325</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_4">
             <property name="minimumSize">
              <size>
               <width>20</width>
               <height>5</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>5</width>
               <height>5</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">background: transparent;
border: 0;
color: transparent;
min-width: 0;</string>
             </property>
             <property name="text">
              <string>Secret</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btnOptions">
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="text">
              <string>Options</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btnContinue">
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="text">
              <string>Continue</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="pageOptions">
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QWidget" name="optionsContent" native="true">
          <layout class="QVBoxLayout" name="verticalLayout_7">
           <item>
            <widget class="QCheckBox" name="commonInstall">
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="text">
              <string>Use common path for BetterDiscord core files</string>
             </property>
             <property name="checked">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="PathChooser" name="installPathChooser" native="true">
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>90</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="commonData">
             <property name="text">
              <string>Use common path for BetterDiscord content</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="PathChooser" name="dataPathChooser" native="true">
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>90</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="autoInject">
             <property name="text">
              <string>Autoinjection routine(attempts to survive Discord updates but could break if Discord changes their structure)</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="optionsFooter" native="true">
          <property name="styleSheet">
           <string notr="true">QPushButton {
	background: transparent;
    color: #fff;
    padding: 7px 10px;
    min-width: 50px;
	background: #17171c;
	border-radius: 5px;
}

QPushButton::hover {
	background: rgb(35, 35, 38);
	background: rgb(62, 204, 156)
}</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="btnCancelOptions">
             <property name="text">
              <string>Cancel</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btnApplyOptions">
             <property name="minimumSize">
              <size>
               <width>70</width>
               <height>28</height>
              </size>
             </property>
             <property name="text">
              <string>Apply</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="pageInstall">
       <layout class="QVBoxLayout" name="verticalLayout_9">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QWidget" name="installContent" native="true">
          <layout class="QVBoxLayout" name="verticalLayout_10">
           <item>
            <widget class="QWidget" name="widget_2" native="true">
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>20</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">border-bottom: 1px solid rgb(62, 204, 156);
color: rgb(62, 204, 156);
margin-left: 65px;</string>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_11">
              <property name="spacing">
               <number>0</number>
              </property>
              <property name="leftMargin">
               <number>0</number>
              </property>
              <property name="topMargin">
               <number>0</number>
              </property>
              <property name="rightMargin">
               <number>0</number>
              </property>
              <property name="bottomMargin">
               <number>0</number>
              </property>
              <item>
               <widget class="QLabel" name="label">
                <property name="text">
                 <string>Products to Remove</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QWidget" name="productsToRemove" native="true">
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_13"/>
            </widget>
           </item>
           <item>
            <widget class="QWidget" name="widget_3" native="true">
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>20</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">border-bottom: 1px solid rgb(62, 204, 156);
color: rgb(62, 204, 156);
margin-left: 65px;</string>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_12">
              <property name="spacing">
               <number>0</number>
              </property>
              <property name="leftMargin">
               <number>0</number>
              </property>
              <property name="topMargin">
               <number>0</number>
              </property>
              <property name="rightMargin">
               <number>0</number>
              </property>
              <property name="bottomMargin">
               <number>0</number>
              </property>
              <item>
               <widget class="QLabel" name="label_4">
                <property name="text">
                 <string>Products to Install</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QWidget" name="productsToInstall" native="true">
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_14"/>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="installFooter" native="true"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>PathChooser</class>
   <extends>QWidget</extends>
   <header>pathchooser.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>captionClose</sender>
   <signal>clicked()</signal>
   <receiver>MainWindowClass</receiver>
   <slot>captionCloseClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>727</x>
     <y>13</y>
    </hint>
    <hint type="destinationlabel">
     <x>372</x>
     <y>243</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>captionHelp</sender>
   <signal>clicked()</signal>
   <receiver>MainWindowClass</receiver>
   <slot>captionHelpClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>655</x>
     <y>13</y>
    </hint>
    <hint type="destinationlabel">
     <x>372</x>
     <y>243</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>captionMin</sender>
   <signal>clicked()</signal>
   <receiver>MainWindowClass</receiver>
   <slot>captionMinClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>691</x>
     <y>13</y>
    </hint>
    <hint type="destinationlabel">
     <x>372</x>
     <y>243</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnOptions</sender>
   <signal>clicked()</signal>
   <receiver>MainWindowClass</receiver>
   <slot>btnOptionsClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>727</x>
     <y>472</y>
    </hint>
    <hint type="destinationlabel">
     <x>424</x>
     <y>249</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnApplyOptions</sender>
   <signal>clicked()</signal>
   <receiver>MainWindowClass</receiver>
   <slot>btnApplyOptionsClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>803</x>
     <y>472</y>
    </hint>
    <hint type="destinationlabel">
     <x>424</x>
     <y>249</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnCancelOptions</sender>
   <signal>clicked()</signal>
   <receiver>MainWindowClass</receiver>
   <slot>btnCancelOptionsClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>727</x>
     <y>472</y>
    </hint>
    <hint type="destinationlabel">
     <x>424</x>
     <y>249</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>commonData</sender>
   <signal>toggled(bool)</signal>
   <receiver>MainWindowClass</receiver>
   <slot>dataCheckboxCheckedChanged(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>424</x>
     <y>153</y>
    </hint>
    <hint type="destinationlabel">
     <x>424</x>
     <y>249</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>commonInstall</sender>
   <signal>toggled(bool)</signal>
   <receiver>MainWindowClass</receiver>
   <slot>installCheckboxCheckedChanged(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>424</x>
     <y>117</y>
    </hint>
    <hint type="destinationlabel">
     <x>424</x>
     <y>249</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnContinue</sender>
   <signal>clicked()</signal>
   <receiver>MainWindowClass</receiver>
   <slot>btnContinueClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>803</x>
     <y>472</y>
    </hint>
    <hint type="destinationlabel">
     <x>424</x>
     <y>249</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>captionCloseClicked()</slot>
  <slot>captionHelpClicked()</slot>
  <slot>captionMinClicked()</slot>
  <slot>btnOptionsClicked()</slot>
  <slot>btnApplyOptionsClicked()</slot>
  <slot>btnCancelOptionsClicked()</slot>
  <slot>installCheckboxCheckedChanged(bool)</slot>
  <slot>dataCheckboxCheckedChanged(bool)</slot>
  <slot>btnContinueClicked()</slot>
 </slots>
</ui>
